code/game/machinery/computer/pod.dm

```diff
@@ -304,14 +304,18 @@
 		visible_message("Cannot locate any mass driver of that ID. Cancelling firing sequence!")
 		return
 
-	if(teleporter_dest)
-		for(var/obj/structure/deathsquad_tele/D in world)
-			if(D.z != src.z)	continue
-			if(D.id_tag == ident_tag)
-				D.icon_state = "tele1"
-				D.ztarget = teleporter_dest
-				D.density = 1
-
+	var/spawn_marauder[] = new()
+	for(var/obj/effect/landmark/L in world)
+		if(L.name == "Marauder Entry")
+			spawn_marauder.Add(L)
+	for(var/obj/effect/landmark/L in world)
+		if(L.name == "Marauder Exit")
+			var/obj/effect/portal/P = new(L.loc)
```

Comment:
I'm not entirely sure how these work, but the original system's intent was; Mech gets launched, hits the `deathsquad_tele`, then flies out of the z-transition with momentum.